### PR TITLE
Adapt jacoco for ci.jenkins.io

### PR DESCRIPTION
**Pull request by timja**
See https://github.com/jenkins-infra/pipeline-library/pull/153#issuecomment-678835664

We could also adapt pipeline-library to add the profile, not big deal

### CODE REVIEW

**Review by oleg-nenashev — COMMENTED** [review]
I would rather keep coverage opt-in. Jacoco may cause significant overheads on developer machines

File: pom.xml [hunk 1]
```diff
@@ -1058,6 +1058,9 @@
     </profile>
     <profile>
       <id>enable-jacoco</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
```
Comment (timja) [inline pom.xml:1063]:
```suggestion
```

**Comment by timja** [comment]
Going to test with just enabling the profile

**Comment by timja** [comment]
tbh don't think this is needed

**Pull request closed without merging**